pom.xml

```diff
@@ -40,6 +40,11 @@
 
   <properties>
     <checkstyle.skip>true</checkstyle.skip>
+    <enforcer.plugin.version>1.4.1</enforcer.plugin.version>
```

Comment:
Could you put a comment here (or below) why the version is selected vs the newer version used later (I assume it is the last version that works on Java 1.6)?

Reply:
Basically because it’s a non final release and I prefer to use a final by default

Reply:
Okay. That's fine.